python/housinginsights/sources/census.py

```diff
@@ -64,38 +64,38 @@ def __init__(self, proxies=None):
                         "acs5_2012"     : "NAME,B01003_001E,B25057_001E,B25058_001E,B25059_001E",
                         "acs5_2013"     : "NAME,B01003_001E,B25057_001E,B25058_001E,B25059_001E",
                         "acs5_2014"     : "NAME,B01003_001E,B25057_001E,B25058_001E,B25059_001E",
-                        "acs5_2015"     : "NAME,B01003_001E,B25057_001E,B25058_001E,B25059_001E",
+                        "acs5_2015"     : "NAME,B01003_001E,B25057_001E,B25058_001E,B25059_001E, B02001_003E, B17020_002E, B19025_001E, B23025_002E, B16008_019E, B09002_015E",
```

Comment:
Ideally, field names should be consistent in all the years and margin of error.. Is their a specific reason why change is limited to acs5_2015?